apps/cnspec/cmd/scan.go

```diff
@@ -135,7 +142,21 @@ var scanCmdRun = func(cmd *cobra.Command, runtime *providers.Runtime, cliRes *pl
 	}
 
 	logger.DebugDumpJSON("report", report)
-	printReports(report, conf, cmd)
+	outputHandler, err := reporter.NewOutputHandler(conf.OutputTarget, conf.OutputFormat)
```

Comment:
I am a bit confused with the output handling here:

1. I would expect that cli output is a proper output handler as well with target STDOUT and format cli
2. This would harmonize the code and also allow us to report to multiple outputs in the future
3. I like that we keep cli out and allow additional output but it would be more explicit if it would be two instances of the output handler.